### DISCUSSION TITLE
stats placeholder please or add /progress

### DISCUSSION THREAD

**Discussion by notiwsy** [Zrips/CMI#7777] [Feature Requests]
can we get the rank requirements placeholder, I want to make a GUI for rank up and I need the amount of required stats
or change the /rankup command system and add /progress
ex:
/rankup : if player is able to rank up cmi will send a message that u need to click the message or something (this is already a feature and I love this) and if player is not able to rankup send a message that they cant. and it has to be customizable so I can change the message to whatever I want
/progress is showing the progress like when they do /rankup with the current cmi version when they are not able to rank up the plugin sends the progress.

**Comment by CloudeLecaw** [comment]
havent you asked the same allready here?

https://github.com/Zrips/CMI/discussions/7760

i liied this old one from you much more because it was contains more picutes etc to see why you requested it. so why you closed it? just because it was one week old?

**Comment by notiwsy** [comment]
idk i thought u didnt see it and cuz I got no responses

**Comment by CloudeLecaw** [comment]
look at the other suggestions it happen often that no Responce is given. this just means that might noone else is interested in it :/

**Reply by notiwsy** [reply]
i am pretty sure placeholder for needed stat is kind of important :o
